### ISSUE TITLE
CANCEL button does not clear the "dirty" flag in Views.

### ISSUE THREAD

**Issue by bd0bd** [backdrop/backdrop-issues#1223]
I do not remember what changes I did there, but CANCEL button does not work in Article Views.
After pressing CANCEL button I get the same warning again and again:

* All changes are stored temporarily. Click Save to make your changes permanent. Click Cancel to               discard your changes.

![cancel](https://cloud.githubusercontent.com/assets/13515302/9936649/a884c61c-5d65-11e5-83b4-994b58ea325a.gif)

**Comment by klonos** [comment]
Confirmed. Here's what I did:

1. Got to Views admin UI (/admin/structure/views)
2. Edit any view (I chose the "Admin Content" one)
3. Make a "harmless" change like add a display. The "dirty" flag fires and the "All changes..." message appears.
4. Hit "Cancel". You are redirected to the main Views UI.
5. Edit the view from step 2 above. The "All changes..." message is still there.
6. Repeat steps 4 & 5. Same thing. Cannot get rid of the message.
